dev-infrastructure/modules/private-endpoint.bicep

```diff
@@ -0,0 +1,68 @@
+param location string
+
+param serviceType string
+param subnetIds array
+
+param privateLinkServiceId string
+
+param groupIds array
+
+param privateEndpointDnsZoneName string
+
+param vnetId string
+
+resource eventGridPrivateEndpointDnsZone 'Microsoft.Network/privateDnsZones@2020-06-01' = {
```

Comment:
remove `eventGrid` prefixes from resources in this module. this is reusable for general private endpoint stuff, right?